annotations/src/test/java/io/github/mkutz/howtotest/annotations/friend/FriendDtoObjectMapperTest.java

```diff
@@ -5,12 +5,12 @@
 
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
 import org.junit.jupiter.api.Test;
+import org.springframework.http.converter.json.Jackson2ObjectMapperBuilder;
 
 class FriendDtoObjectMapperTest {
 
-  static ObjectMapper objectMapper = new ObjectMapper().registerModule(new JavaTimeModule());
+  static ObjectMapper objectMapper = Jackson2ObjectMapperBuilder.json().build();
```

Comment:
One might consider this as an alternative way to get a preconfigured `objectMapper`. But it requires spring-web to be on the test classpath. See also [Javadoc](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/http/converter/json/Jackson2ObjectMapperBuilder.html)